plugins/woocommerce/tests/php/src/Internal/Admin/Agentic/AgenticWebhookPayloadBuilderTest.php

```
@@ -0,0 +1,216 @@
+	 *
+	 * @var AgenticWebhookPayloadBuilder
+	 */
+	protected $payload_builder;
+
+	/**
+	 * Set up before each test.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		// Use the WooCommerce logger for tests.
+		$logger                = wc_get_logger();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$logger'. (undefined)

(UnusedLocalVariable)